### ISSUE TITLE
0474IbnKhalafBaji.MuntaqaMinSharhMuwatta

### ISSUE THREAD

**Issue by Maroussia** [OpenITI/Annotation#1805]
**LINK:** <https://raw.githubusercontent.com/OpenITI/0475AH/master/data/0474IbnKhalafBaji/0474IbnKhalafBaji.MuntaqaMinSharhMuwatta/0474IbnKhalafBaji.MuntaqaMinSharhMuwatta.Shamela0006684-ara1>
**ISSUE:** On PageV01P130, the text starting with {### | [المنتقى]} and ending with {~~وجه الوجوب في الجملة لأجل المشقة والله أعلم وأحكم.} belongs to the previous section and should come before the hadith about the adhan that is mentioned just before, following the two editions of the texts. See the [Egyptian edition on which the digital version is based, p. 130](https://archive.org/details/MuntaqaBaji/01/page/n129/mode/2up); and the [Lebanese more modern edition, p.2, section 143](https://archive.org/details/MontaqaCharhMowattaeMalek/mntqshmutta2/page/n1/mode/2up).
**RESOLUTION:** The digital text should probably follow the edition, except if one can consult the manuscript and check whether the error comes from there.

**NOTE:** Please, paste the URI of the text into the header.

**Comment by pverkind** [comment]
The text is indeed horrible. Shamela seems to have converted it from a digitized version that presented the original text of the Muwatta and the sharh separately (all sections of the original text (?), marked with "ṣād", are at the top of the page, or mostly so, and the sharḥ (?), marked with shīn, follow), and something must have gone wrong in the conversion. I have taken the following steps to solve this: 
1. I took the improved text from a more recent Shamela scrape; it still has some problems, but not the many html tags: 0474IbnKhalafBaji.MuntaqaMinSharhMuwatta.Sham19Y0006684-ara1.inProgress
2. I started manually correcting the order of the paragraphs, but then realized this was a 7-volume work. 
3. I wrote a script that attempts to correct the order
4. I have run a number of tests (see the YML file) to find mistakes that were left uncorrected (or even introduced) by the script. This process is not finished; will continue after the vacation.
